build-tools/packages/build-cli/src/test/commands/vnext/check/latestVersions.test.ts

```diff
@@ -0,0 +1,140 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { EOL } from "node:os";
+import { runCommand } from "@oclif/test";
+import { expect } from "chai";
+import { afterEach, describe, it } from "mocha";
+import mockedEnv from "mocked-env";
```

Comment:
You're not using mockedEnv in this file, really. You can remove this line and the uses below - it's not needed because you're not testing that flag values are ready from environment variables, which is what mockedEnv is used for.